tests/sortition.rs

```diff
@@ -17,25 +17,35 @@ fn test_deterministic_sortition_1() {
     // Create provisioners with bls keys read from an external file.
     let mut p = generate_provisioners(5);
 
+    let committee_size = 64;
+
     // Execute sortition with specific config
     let cfg = Config::new(Seed::default(), 1, 1, 64);
     p.update_eligibility_flag(cfg.round);
 
+    let committee = Committee::new(PublicKey::default(), &mut p, cfg);
+
     assert_eq!(
-        vec![1, 2, 1],
-        Committee::new(PublicKey::default(), &mut p, cfg).get_occurrences()
+        committee_size,
+        committee.get_occurrences().iter().sum::<usize>()
     );
+    assert_eq!(vec![7, 23, 13, 21], committee.get_occurrences());
 }
 
 #[test]
 fn test_deterministic_sortition_2() {
     // Create provisioners with bls keys read from an external file.
     let mut p = generate_provisioners(5);
 
-    let cfg = Config::new(Seed::from([3u8; 48]), 7777, 8, 45);
+    let committee_size = 45;
```

Comment:
Does this test still make sense?
What's the point of testing a committee of size 45?
If anything, this test should fail, assuming the committee should always have size 64 (or 1 in Selection).

Reply:
This is a test for the deterministic sortition. It's a lower-level... it doesn't know anything about how the committee is used by the consensus itself.

This test, as the other two in the same file, only check that the count of the credits extracted is the same of the committee_size whether the number of provisioners is greater, less, or equal